code/modules/mob/living/simple_animal/abnormality/teth/crumbling_armor.dm

```diff
@@ -0,0 +1,94 @@
+/mob/living/simple_animal/hostile/abnormality/crumbling_armor
+	name = "Crumbling Armor"
+	desc = "A thoroughly aged suit of samurai style armor with a V shaped crest on the helmet. It appears desuetude."
+	icon = 'ModularTegustation/Teguicons/32x48.dmi'
+	icon_state = "crumbling"
+	maxHealth = 600
+	health = 600
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(50, 50, 55, 55, 60),
+		ABNORMALITY_WORK_INSIGHT = 40,
+		ABNORMALITY_WORK_ATTACHMENT = 0,
+		ABNORMALITY_WORK_REPRESSION = list(60, 60, 65, 65, 70)
+			)
+	pixel_y = -8
+	base_pixel_y = -8
+	work_damage_amount = 5
+	work_damage_type = RED_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/daredevil,
+		/datum/ego_datum/armor/daredevil
+		)
+
+	var/buffIcon = 'ModularTegustation/Teguicons/32x48.dmi'
+	var/mob/living/carbon/human/courageous = list()
+	var/mob/living/carbon/human/reckless = list()
+	var/mob/living/carbon/human/foolish = list()
+	var/mob/living/carbon/human/suicidal = list()
```

Comment:
Looks like a waste of space.
Make it an associative list instead.

Reply:
> The overlay should be on the layer below mob, so it doesn't look that weird. Sprite itself is rather weird, too.

I wanted a better sprite for it but I was told "it was fine". I'll request one in Spriter General